grapher/stackedCharts/MarimekkoChart.tsx

```diff
@@ -761,7 +764,17 @@ export class MarimekkoChart
         const strokeColor =
             isHovered || isSelected ? "#555" : isPlaceholder ? "#ccc" : "#666"
         const strokeWidth = isHovered || isSelected ? "1px" : "0.5px"
-        const strokeOpacity = isPlaceholder ? 0.2 : 1.0
+        const strokeOpacity = isPlaceholder ? 0.8 : isSelected ? 0.5 : 0.1
+        const fillOpacity = isHovered
+            ? 0.7
+            : isFaint
+            ? 0.2
+            : isSelected
+            ? isPlaceholder
+                ? 0.3
+                : 0.7
+            : 0.7
```

Comment:
I find these long tertiaries hard to read, and maybe some consolidation doesn't hurt here anyway.
There seem to be three levels going on:
* 0.2 (faint)
* 0.3 (normal)
* 0.7 (highlighted)

Maybe just introduce a `const isHighlighted = isHovered || (isSelected && !isPlaceholder)` (or similar)?